pkg/cmd/secret/set/set_test.go

```diff
@@ -204,6 +216,46 @@ func Test_setRun_repo(t *testing.T) {
 	assert.Equal(t, payload.EncryptedValue, "UKYUCbHd0DJemxa3AOcZ6XcsBwALG9d4bpB8ZT0gSV39vl3BHiGSgj8zJapDxgB2BwqNqRhpjC4=")
 }
 
+func Test_setRun_env(t *testing.T) {
```

Comment:
This is almost like `Test_setRun_repo` above, except the 2nd mock and the `EnvName` option... Might be better to parametrize the existing test?

Reply:
I think this is fine for now!